src/core/pipe.py

```
@@ -0,0 +1,62 @@
+        self.processor = AudioProcessor(config_processor)
+        self.config_asr = config_asr
+        self.is_warmed_up = False
+
+    def warmup(self) -> None:
+        """
+        Warm up the pipeline
+        """
+        self.asr.warmup(1, self.processor.sr)
+        self.is_warmed_up = True
+
+    def __call__(self, audios: list[np.ndarray]) -> list[list[Replica]]:
```

Pylint (3.3.7)

[refactor] Too many local variables (17/15)

(R0914)